addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8646,8 +8661,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Edit settings"
```

Comment:
@da-anda Please keep in mind which entry appears in which context menus. "New timer" appaers not in context menus auf timers, but in context menus of epg events (in search window and guide window, not timers window). Therefore it is essential to have the "timer" in the label and ofc to have the menu entry itself.

Regarding "Edit" vs. "Edit settings": Would be fine be me. Maybe I change this in one of my next PRs.

Rearding "Find similar": ... EPG events, but this string is not end user "compatible". If you come up with something better I will be ahppy to change it.